auth_otp/management/commands/cleanup_otp.py

```
@@ -0,0 +1,102 @@
+        token_count = self._cleanup_tokens(dry_run)
+        
+        # نمایش خلاصه
+        self.stdout.write(
+            self.style.SUCCESS(
+                f'\nخلاصه پاکسازی:\n'
+                f'- OTP های حذف شده: {otp_count}\n'
+                f'- توکن‌های حذف شده از blacklist: {token_count}'
+            )
+        )
+    
+    def _cleanup_otps(self, dry_run, age_hours):
```

Ruff (0.12.2)

Missing return type annotation for private function `_cleanup_otps`

(ANN202)